examples/hello-world-udp-proxy/dvc-proxy-config.json

```
@@ -0,0 +1,10 @@
+{
+  "instances": [
+    {
+      "sdkKey": "${DEVCYCLE_SDK_KEY}",
+      "unixSocketPath": "/tmp/dvc_lb_proxy.sock",
+      "unixSocketEnabled": true,
```

```suggestion
      "unixSocketEnabled": true,
      "unixSocketPermissions": 777,
```

You'll need to set the file perms here too.

---

I've removed the extra commands from the supervisord configuration and I'm testing this functionality now

---

@JamieSinn Is this configuration option supposed to negate the need to alter the socket file permissions after the proxy command is run?

---

yep!

---

I'm not able to get the actual variable value when using this configuration option and removing the extra commands for altering the socket file permissions, which is why I ask.

---

Hmm - that's odd - let me do a quick test.

---

I'll push up the changes I used to test that, I think that'd help, and I can resolve any issues afterward.

---

sounds good. I have a feeling this might be on my implementation.

---

Yep - looks like that was on me. I fixed it to now support octal perms. 1.1.1 will fix it